### PR TITLE
Feature hit api

### PR DESCRIPTION
**Pull request by kosher9**
- Implemented submit action
- Implemented refresh action
- Used `async` and `await` JavaScript features to consume the API

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1 +1,63 @@
...
+
+const BASE_URL = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/';
+
+const GAME_ID = '0wmZGRhOtcZ3YyoWhvRi';
+
+async function getScores() {
+  const response = await fetch(`${BASE_URL}games/${GAME_ID}/scores/`);
+  const data = await response.json();
+  return data;
+}
```
Comment (medaminedev66) [inline src/index.js]: - [x] While you use ES6, It would be great to use arrow functions instead of the function keyword.

File: src/index.js [hunk 2]
```diff
@@ -1 +1,63 @@
...
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify(newScore),
+  });
+  const data = await response.json();
+  refreshTable();
+  return data;
+}
```
Comment (medaminedev66) [inline src/index.js]: - [x] Kindly try to use arrow functions instead of the function keyword.

**Review by medaminedev66 — CHANGES_REQUESTED** [review]
Hi @kosher9  😃, :wave:, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- the app runs as expected. The user can submit user data and it reflects on the scoreboard. great job 👍.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by BenMukebo — APPROVED** [review]
### Hi @kosher9, 👋 

# STATUS: Approved  🎉 

Great work on making the changes requested by a previous reviewer :1st_place_medal:  
 Your project is complete!  There is nothing else to say other than... it's time to merge it
Congratulations! 🎉  🎉 🎉 




## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- [OPTIONAL] Kindly, Add a descriptive README file to your project, it will be great to add  instructions on how to run the project. This includes `npm install` , `npm start` etc

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @BenMukebo ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**